projects/project-player-library/src/lib/pages/preview-details-page/preview-details-page.component.html

```diff
@@ -0,0 +1,69 @@
+
+    <mat-card class="max-w-full p-4 mt-0 mx-2 mb-3" style="box-shadow: rgba(133, 119, 119, 0.555) 5px 5px 4px ; border-radius: 20px !important;">
```

Comment:
Make the changes which was added for the details-page comments in this page too

Reply:
the preview page is just to show the template and the actions will be performed depending upon the start improvement

Reply:
I meant the changes like @for, @if and the other comments

Reply:
I don't see any changes done here